.github/workflows/gpu_4090_full_benchmark.yml

```diff
@@ -24,7 +24,9 @@ on:
 jobs:
   cuda-integer-benchmarks:
     name: Cuda integer benchmarks for all operations flavor  (RTX 4090)
-    if: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || contains(github.event.label.name, '4090_bench') }}
+    if: ${{ github.event_name == 'workflow_dispatch' ||
+      github.event_name == 'schedule' && github.repository == 'zama-ai/tfhe-rs' ||
+      contains(github.event.label.name, '4090_bench') }}
```

Comment:
Actually we should do this on all bench workflows, not just the rtx one

Reply:
I did not see non RTX benches on internal